### DISCUSSION TITLE
Is there a way to stop Crashplan from checking for updates?

### DISCUSSION THREAD

**Discussion by greggerca** [jlesage/docker-crashplan-pro#456] [Q&A] [answered]
I'm not sure if the constant checking for updates (between container releases) is stopping backup progress or not, but is there a setting to say "don't try to upgrade Crashplan"?

**Comment by jlesage** [comment, marked as answer]
To my knowledge this is no such option.
If the CrashPlan client is tool old, the backup might stop functioning properly.

**Reply by greggerca** [reply]
Thank you!
I guess there must be some new version of it available. I am on your current image, and it seems like things are still working. It just tries to upgrade every hour.

**Reply by jlesage** [reply]
I don't see update on my side.  What version of CrashPlan do you have currently ?

**Reply by greggerca** [reply]
(Update)
Yes - operator error. I may not have waited long enough for the Action -> Reset to complete last time!
It is 11.2.1.23 now.
Thank you!

(Original)
Oh, it says in the About menu: 11.0.1.33 - which seems low (the download site seems to offer 11.2.x.y).

I can try re-downloading the "latest" branch and clearing/restarting the container.
Probably operator error.
Thank you!
